src/js/themes/base.js

```diff
@@ -1259,6 +1259,49 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         },
       },
     },
+    pageHeader: {
+      subtitle: {
+        // any paragraph props
+        margin: 'none',
+        fill: true,
+      },
+      title: {
+        // any heading props
+        size: 'small',
```

Comment:
small makes sense for HPE theme, but thinking Grommet should use the default heading size.

```suggestion
```

Reply:
Done.